drivers/SmartThings/zigbee-button/src/zigbee-multi-button/SLED/init.lua

```diff
@@ -0,0 +1,93 @@
+-- Copyright 2022 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local capabilities = require "st.capabilities"
+local clusters  = require "st.zigbee.zcl.clusters"
+local device_management = require "st.zigbee.device_management"
+local log = require "log"
+local Level = clusters.Level
+local OnOff = clusters.OnOff
+local ColorControl = clusters.ColorControl
+
+local fields = {
+  IGNORE_MOVETOCOLORTEMP = "ignore_next_movetocolortemperature"
+}
+
+local emit_pushed_event = function(button_name, device)
+  local additional_fields = {
+    state_change = true
+  }
+  local event = capabilities.button.button.pushed(additional_fields)
+  local comp = device.profile.components[button_name]
+  if comp ~= nil then
+    device:emit_component_event(comp, event)
+    device:emit_event(event)
```

Comment:
the main component does not have the button capability so this will fail, same below

Reply:
Do you mean to delete this code?
Or do you want me to add button from main in profile?
-id: main
    capabilities:
      - id: button
        version: 1

Reply:
Up to you how to implement it. Either of those solutions would work.

In our drivers, we have had the main component of a multi-button device emit an event when _any_ of the buttons are pressed, but some users have found this confusing.

Reply:
We will also be confused if we have a main component.
Then please guide me on how to modify the code without main component.
If not, I will add main component.

Reply:
just remove the `device:emit_event(event)` line

That attempts to emit the event from the main component, which does not have the capability used by the event, and so it would log a warning.

Reply:
I will check it after modifying it. Thanks.

Reply:
I modified the code, but the held button is not visible, so I am rechecking it.